Server/package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file should not be here. (I'm not really sure how it was created either)

Reply:
Ok I removed it.